testkit/src/main/java/org/apache/calcite/test/CalciteAssert.java

```diff
@@ -679,6 +679,7 @@ static void assertPrepare(
       Connection connection,
       String sql,
       boolean materializationsEnabled,
+      List<Pair<Hook, Consumer>> hooks,
```

Comment:
This is also a public API (testkit are expected to be used by third party projects)

Reply:
Added an overload function to maintain the publish API.